vta/src/de10nano/cma_api.h

```diff
@@ -27,6 +27,8 @@
 extern "C" {
 #endif
 
+#include <stddef.h>
```

Comment:
GCC 7.4.0, the default in my UBUNTU 18.04.3 distro on the DE10-Nano exits with error without it because of undefined **size_t** type.
You may be using an older version that is more lenient in allowing not specifying required headers.

Reply:
Thanks, that make total sense.